### DISCUSSION TITLE
Raspberry Pi Clients

### DISCUSSION THREAD

**Discussion by seancarte** [ltsp/ltsp#227] [Q&A]
I am looking at using Raspberry Pi 3B+  boards as clients but have run into difficulties and was wondering if anyone could help.

My initial thinking was that I could just PXE-boot the Pi and it would act like a standard client, but that didn't work as expected. I found some documentation (https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/net_tutorial.md) that suggests that the Pi should be able to network boot ($ vcgencmd otp_dump | grep 17: does produce the correct output, 17:3020000a, and I am able to boot from a USB drive). But plugged into a network without a SD card, nothing happens.

I also found this documentation: https://help.ubuntu.com/community/UbuntuLTSP/RaspberryPi and tried following it to add the required image to an existing LTSP5 server, but ran into two problems:
the LTSP build ended in errors, and
the RPi still seems to want to boot from the SD card; it doesn't attempt to use the network.

I am about to start from scratch with a new server. Has anybody got any advice on how to proceed? Would I be better off using the latest LTSP, or should I continue with LTSP5 on Ubuntu 16.04? Is anybody using RPis?

**Comment by alkisg** [comment]
Hi, support for Raspberry Pis is discussed there:
https://github.com/ltsp/ltsp/issues/68

Please copy/paste your question there!
